bundles/org.openhab.ui/web/src/js/openhab/utils.js

```diff
@@ -3,7 +3,12 @@ import diacritic from 'diacritic'
 
 export default {
   normalizeLabel: (label) => {
-    return diacritic.clean(label.normalize('NFKD')).replace(/\s+/g, '_').replace(/[^0-9a-z_]/gi, '')
+    return diacritic.clean(label.normalize('NFKD')).replace(/\s+/g, '_').replace(/[^0-9a-z_]/gi, '').replace(/^([0-9])/, '_$1')
```

Comment:
```suggestion
    return diacritic.clean(label.normalize('NFKD')).replace(/\s+/g, '_').replace(/[^0-9a-z_]/gi, '').replace(/^([0-9-])/, '_$1')
```

Reply:
The previous regex removes `-` so this is not necessary here.

However, I might need to have two versions of this,  one specific for thinguid that allows dashes.

Reply:
I've added a separate check for thing/channel id